src/lib.rs

```diff
@@ -112,13 +112,12 @@
 #![cfg_attr(feature = "nightly", feature(test))]
 #![cfg_attr(feature = "nightly", feature(concat_idents))]
 #![cfg_attr(feature = "nightly", feature(core_intrinsics))]
-#[cfg(feature = "nightly")]
-extern crate test;
 
+extern crate alloc;
 #[cfg(test)]
 extern crate std;
-
-extern crate alloc;
+#[cfg(feature = "nightly")]
+extern crate test;
```

Comment:
Do we even need this any more after the move to criterion?

Reply:
Nice catch! Fixed.